tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1233,18 +1245,18 @@ error: \\"x\\" is not initialized
    |                  ^^^^^^^^^^ \\"x\\" is not initialized
 
 
-error: struct fields must be immutable got: MutArray<str>
+error: Struct fields must have immutable types
    --> ../../../examples/tests/invalid/structs.w:20:3
    |
 20 |   f: MutArray<str>;
-   |   ^ struct fields must be immutable got: MutArray<str>
```

Comment:
The type annotation is required and is right next to the field name, so removing the duplication in the error message seemed like a net good, I think?

Reply:
yes